src/components/Dashboard/Dashboard.tsx

```diff
@@ -32,9 +38,39 @@ export const Dashboard = withRouter(props => {
   /**
    * Set token in token context
    */
-  tokenContext.setToken(getCookie("sessionToken"));
+  useEffect(() => {
+    /**
+     * Set token in token context
+     */
+    tokenContext.setToken(getCookie("sessionToken"));
+    console.log("setToken called!", tokenContext.token);
+  }, []);
 
-  console.log("setToken called!", tokenContext.token);
+  useEffect(() => {
+    console.log("Enter use effect with toker", tokenContext.token);
+    if (tokenContext.token) {
+      const url =
+        customWindow._env_.IO_ONBOARDING_PA_API_HOST +
+        ":" +
+        customWindow._env_.IO_ONBOARDING_PA_API_PORT +
+        "/profile";
+      fetch(url, {
+        headers: {
+          Accept: "application/json",
+          Authorization: "Bearer " + tokenContext.token
+          // 'Content-Type': 'application/json'
+        },
+        method: "GET"
+      })
+        .then(response => {
+          return response.json();
+        })
+        .then(responseData => {
+          // TODO: aggiungere codice per gestione risposta servizio profile
```

Comment:
please add a pivotal task reference (may we have the comment in english ?)

Reply:
fixed comment in english and added pivotal task reference